dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/listener/AggregateDatasetSMSListener.java

```diff
@@ -0,0 +1,191 @@
+package org.hisp.dhis.sms.listener;
+
```

Comment:
The license text is missing

Reply:
Yeah I had an outstanding PR with the licensing and styling updates, it's merged now. I also added the license to some existing files that didn't have it.